ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -203,7 +204,7 @@ public void onResume() {
   public void requestBannerAd(Context context, MediationBannerListener mediationBannerListener,
       Bundle serverParameters, AdSize adSize, MediationAdRequest mediationAdRequest,
       Bundle mediationExtras) {
-    vungleBannerAdapter = new VungleBannerAdapter(context, VungleInterstitialAdapter.this,
+    vungleBannerAdapter = new VungleBannerAdapterProxy(context, VungleInterstitialAdapter.this,
```

Comment:
could we swap names VungleBannerAdapterProxy  and VungleBannerAdapter to have fewer changes in AdMob Pr?

Reply:
yes, done.